symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php

```diff
@@ -76,7 +76,7 @@ public function checkForPunchOutOverLappingRecords($punchInTime, $punchOutTime,
             $records1 = $query1->execute();
 
          if ((count($records1) == 1) && ($records1[0]->getId() == $recordId)) {
-                
+
```

Comment:
Why empty block ?

Reply:
Previous codes of this file have re-formatted due to my mistake